github-dark.css

```diff
@@ -1529,8 +1529,9 @@
   a.filter-item.selected, .date-selector td.selected,
   .date-selector td.selectable-day:hover, .subnav-item:hover, .subnav-item:focus,
   .owner-select-target:hover, .owner-select-target:focus,
-  .suggester li.navigation-focus, .suggester li.navigation-focus small {
-    color: #efefef !important;
+  .suggester li.navigation-focus, .suggester li.navigation-focus small,
+  .label-select-menu .select-menu-item.selected, octicon-x {
```

Comment:
It appears to be (heres a demo with a purple color for added effect)

it also works with ```.octicon .octicon-x``` which is what the actual class said

if its right or wrong... you tell me ;)

![demo-a](https://cloud.githubusercontent.com/assets/3521959/17359116/1461951a-595d-11e6-82a8-15ba62fdec80.gif)